config/i18n-tasks.yml

```diff
@@ -93,9 +93,9 @@ ignore_unused:
 - 'simple_form.{yes,no}'
 - 'simple_form.{placeholders,hints,labels}.*'
 - 'simple_form.{error_notification,required}.:'
-- 'admin.*.index.header'
+- 'admin.*.*.header'
```

Comment:
@dariusf I changed this because it appears that sometimes we use other pages, e.g. the course users page.